website/source/docs/providers/aws/r/elastic_transcoder_pipeline.html.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: "aws"
+page_title: "AWS: aws_elastictranscoder_pipeline"
+sidebar_current: "docs-aws-resource-elastic-transcoder"
+description: |-
+  Provides an Elastic Transcoder pipeline resource.
+---
+
+# aws\_elastictranscoder\_pipeline
+
+Provides an Elastic Transcoder pipeline resource.
+
+## Example Usage
+
+### Elastic Transcoder Pipeline
+
+```
+resource "aws_elastictranscoder_pipeline" "bar" {
+  input_bucket = "${aws_s3_bucket.input_bucket.bucket}"
+  name         = "aws_elastictranscoder_pipeline_tf_test_"
```

Comment:
yes, when I was avoiding my own name collisions ;) (I can remove it now if though)